src/react/components/pages/connections/connectionForm.tsx

```diff
@@ -85,6 +86,12 @@ export default class ConnectionForm extends React.Component<IConnectionFormProps
                     </span>
                 </h3>
                 <div className="m-3 text-light">
+                    { isBrowser() && this.state.providerName === "bingImageSearch" &&
+                    <div className="alert alert-warning" role="alert">
+                        <i className="fas fa-exclamation-circle" aria-hidden="true"></i>
```

Comment:
You can use bootstrap build in classes like `mr-1` which will add margin-right to the element vs defining a new css class.

Reply:
ah, nice, will do.